examples/dummy_plugin/pyproject.toml

```diff
@@ -10,7 +10,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-importlib-metadata = {version = "~3.4.0", python = "<3.8"}
+importlib-metadata = {version = "<3", python = "<3.8"}
```

Comment:
Why this change? Seems undesirable to move back to an older version of importlib-metadata.